src/main/java/com/icssociety/automatedrequests/SetHeader.java

```diff
@@ -0,0 +1,51 @@
+package com.icssociety.automatedrequests;
+
+import org.javalite.activejdbc.*;
+
+import de.sstoehr.harreader.HarReader;
+import de.sstoehr.harreader.HarReaderException;
+import de.sstoehr.harreader.model.Har;
+import de.sstoehr.harreader.model.HarEntry;
+import de.sstoehr.harreader.model.HarHeader;
+
+import java.util.*;
+import java.io.*;
+
+public class SetHeader {
+    
+	public static void main(String[] args) throws HarReaderException {
```

Comment:
You may want to move all of this code into a different function other than main. Perhaps public static void save()